templates/index.html

```diff
@@ -853,6 +853,10 @@ <h4 class="modal-title">העדפות:</h4>
                 <label for="checkbox-type-pedestrian" class="btn one">אחר</label>
                 <input type="radio" name="acctype" id="checkbox-type-else" value=1 onclick="app.loadFilter()" />
                 <label for="checkbox-type-else" class="btn one">פגיעה בהולך רגל</label>
+                <input type="radio" name="acctype" id="checkbox-type-bike-no-cas" value=21 onclick="app.loadFilter()" />
+                <label for="checkbox-type-bike-no-cas" class="btn one">אופניים ללא נפגעים</label>
+                <input type="radio" name="acctype" id="checkbox-type-with-cas" value=22 onclick="app.loadFilter()" />
+                <label for="checkbox-type-with-cas" class="btn one">אופניים עם נפגעים</label>
```

Comment:
You forgot the `bike` in the second radio button. Also, any reason not to write the whole word `casualty`? Or maybe `injured` would be more accurate ("casualty" entails death, at least in my mind).